rosdep/python.yaml

```diff
@@ -795,6 +795,12 @@ python-debian:
     wily_python3: [python3-debian]
     xenial: [python-debian]
     xenial_python3: [python3-debian]
+    yakkety: [python-debian]
+    yakkety_python3: [python3-debian]
+    zesty: [python-debian]
+    zesty_python3: [python3-debian]
+    artful: [python-debian]
```

Comment:
Thanks for the quick addition. Though artful needs to go at the top of the list due to alphabetical ordering.